repo_evaluate/main.py

```diff
@@ -103,7 +103,7 @@ def get_repo_addresses(file_location: str) -> list[str]:
 
             match BUILD_TOOLS[repo]:
                 case "Maven":
-                    if build.validate_maven_pom(str(BUILD_FILES[repo]), "./resources/maven-4.0.0.xsd"):
+                    if build.validate_maven_pom(str(BUILD_FILES[repo]), "repo_evaluate/resources/maven-4.0.0.xsd"):
```

Comment:
Ideally, rather than assuming the program is run in a specific directory, the resources should be obtained relative to the program's installation directory.

Reply:
This would work irrespective to where the program is installed. In order to run the program, the poetry instance will be in the correct place to find the resources (which are part of the build folder structure)